x-pack/plugins/security/public/ui_api/components.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC, PropsWithChildren, PropsWithRef } from 'react';
+import React from 'react';
+
+import type { CoreStart } from 'src/core/public';
+
+import { getChangePasswordComponent } from '../account_management/change_password/change_password_async';
+import { getPersonalInfoComponent } from '../account_management/personal_info/personal_info_async';
```

Comment:
__nit:__ (once you propagate all required exports through relevant `index.ts` files)
```suggestion
import { getChangePasswordComponent,  getPersonalInfoComponent } from '../account_management';
```

Reply:
I tried to did it before, but it resulted in an increased bundle size.

I took the following info from `security/target/public/metrics.json` after running the first command from [CI Metrics doc](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html#ci-metric-validating-limits).

Before:
```
{
  "group": "page load bundle size",
  "id": "security",
  "value": 87166,
  "limit": 95864,
  "limitConfigPath": "packages/kbn-optimizer/limits.yml"
}
```
  
After:
```  
{
  "group": "page load bundle size",
  "id": "security",
  "value": 101392,
  "limit": 95864,
  "limitConfigPath": "packages/kbn-optimizer/limits.yml"
}
```

I'm guessing that once async components are exported through the index file and we use that index file somewhere that also brings all the sync components' dependencies and increases bundle size.

I kept the imports as is, but let me know if you have more thoughts.

Reply:
Good idea, absolutely.

Reply:
Added a comment here: https://github.com/elastic/kibana/pull/99791/commits/1880755587bfed1c337da8cfc626999e52569041